README.md

```diff
@@ -247,4 +247,26 @@ virsh snapshot-create-as --domain vm_name --name snapshot_2
 virsh snapshot-list vm_name
 # and revert to the first one
 virsh snapshot-revert vm_name --snapshotname snapshot_1
-```
\ No newline at end of file
+```
+
+### Development
+install libvirt-dev package:
+
+Debian:
+```shell
+apt-get install libvirt-dev
```

Comment:
On Fedora, the package is called libvirt-devel. We also need python3-devel, and I suspect we need this one on Debian too to be able to prepare the venv.

Reply:
https://github.com/virt-lightning/virt-lightning/commit/c47dac07ea8d1fb3cbefbf02e13bf9525cc21401 here is recently updated related document. Could you rebase your patch, or back-merge master to your branch to resolve?

Reply:
My bad, didn't test on nice clean easily created VM... luckily there is a tool for that.

Seems there are a few other common dependency's required, i'll track them down and update.